### DISCUSSION TITLE
【Roadmap】版本迭代的RoadMap

### DISCUSSION THREAD

**Discussion by lxw15337674** [tangly1024/NotionNext#2138] [Ideas]
**为什么提出这个新的特性改动**
方便其他人帮忙贡献代码。

**描述一下你推荐的解决方案**
根据四象限法则列出后面要做的feature。

**描述一下你考虑过的其它替代解决方案**
1.参考https://github.com/umijs/qiankun/discussions/2688
2.参考https://github.com/flutter/flutter/wiki/Roadmap
**补充说明**
补充与此特性相关的内容

**Comment by tangly1024** [comment]
很棒的提议，我尝试模仿着做一个

- https://github.com/tangly1024/NotionNext/wiki/Roadmap
- https://github.com/tangly1024/NotionNext/discussions/2140

不够完美，却是一个开始。
